sys/include/ubjson.h

```diff
@@ -187,14 +187,15 @@ typedef enum {
     UBJSON_SIZE_ERROR,        /**< the length of a field exceeded SSIZE_MAX */
 } ubjson_read_callback_result_t;
 
```

Comment:
@jcarrano Should we add a deprecation warning to the documentation as well?

Reply:
yes we should, I did not think abiout that.